diff --git a/m4/guestfs_misc_libraries.m4 b/m4/guestfs_misc_libraries.m4
index 9b22670..3f81396 100644
--- a/m4/guestfs_misc_libraries.m4
+++ b/m4/guestfs_misc_libraries.m4
@@ -50,7 +50,7 @@ AC_ARG_WITH([readline],[
 LIBREADLINE=
 AS_IF([test "x$with_readline" != xno],[
     AC_CHECK_LIB([readline], [main],
-        [AC_SUBST([LIBREADLINE], ["-lreadline -lncurses"])
+        [AC_SUBST([LIBREADLINE], ["-lreadline -lncurses $LIBTINFO_LIBS"])
          AC_DEFINE([HAVE_LIBREADLINE], [1],
                    [Define if you have libreadline.])
         ],
